### PR TITLE
homepage-updated

### PR DESCRIPTION
**Pull request by dancantor**

### CODE REVIEW

File: wist-scoreboard-generator/src/index.html [hunk 1]
```diff
@@ -6,6 +6,9 @@
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```
Comment (vladcantor) [inline wist-scoreboard-generator/src/index.html:10]: are these different from the ones in the styles.scss file? You should use only one font in the entire application.

File: wist-scoreboard-generator/src/app/app-routing.module.ts [hunk 2]
```diff
@@ -1,7 +1,12 @@
+import { HomepageComponent } from './Pages/homepage/homepage.component';
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { GameSettingsPageComponent } from './Pages/game-settings-page/game-settings-page.component';
 
-const routes: Routes = [];
+const routes: Routes = [
+  {path:'', component: HomepageComponent},
+  {path:'game', component: GameSettingsPageComponent}
```
Comment (vladcantor) [inline wist-scoreboard-generator/src/app/app-routing.module.ts]: I would redirect any non matching path to the home page. see: https://angular.io/guide/router#setting-up-redirects

File: wist-scoreboard-generator/src/app/Pages/homepage/homepage.component.scss [hunk 3]
```diff
@@ -0,0 +1,69 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap');
```
Comment (vladcantor) [inline wist-scoreboard-generator/src/app/Pages/homepage/homepage.component.scss]: is this needed if you import it in styles.scss?

File: wist-scoreboard-generator/src/app/Pages/homepage/homepage.component.html [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<div class="container">
+    <img src="../../../assets/img/pexels-dids-2508565.jpg">
```
Comment (vladcantor) [inline wist-scoreboard-generator/src/app/Pages/homepage/homepage.component.html]: is this working if you build the angular application with ng build --prod?

File: wist-scoreboard-generator/src/app/Pages/homepage/homepage.component.html [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+    <div class="title">
+        Wist Scoreboard Generator
+        <div class="description">
+            Genereaza tabele de scor pentru jocul de carti Wist. 
+            Tot ce trebuie sa faci este sa introduci jucatorii si optiunile de joc.
```
Comment (vladcantor) [inline wist-scoreboard-generator/src/app/Pages/homepage/homepage.component.html]: You shouldn't build applications in Romanian. Please use https://angular.io/guide/i18n-overview and develop the application English first, but with support for Romanian

File: wist-scoreboard-generator/package.json [hunk 6]
```diff
@@ -11,10 +11,12 @@
   "private": true,
   "dependencies": {
     "@angular/animations": "~12.1.0-",
+    "@angular/cdk": "^12.2.13",
```
Comment (vladcantor) [inline wist-scoreboard-generator/package.json]: please do an `ng update @angular/cli@^12 @angular/core@^12`. See https://angular.io/cli/update for more details. The version of the packages used are pretty old.

Comment (vladcantor) [reply wist-scoreboard-generator/package.json]: Just an FYI the current version of Angular is 13 but angular 12 has support until November. See: https://angular.io/guide/releases#support-policy-and-schedule

**Review by vladcantor — APPROVED** [review]
You have done a great job :) Congrats!

File: wist-scoreboard-generator/src/app/Pages/homepage/homepage.component.html (hunk 4 above)
Comment (dancantor) [reply wist-scoreboard-generator/src/app/Pages/homepage/homepage.component.html]: Yes, I think it does. Assets folder is inside the angular project, so I think it is  fine

File: wist-scoreboard-generator/src/app/Pages/homepage/homepage.component.scss (hunk 3 above)
Comment (dancantor) [reply wist-scoreboard-generator/src/app/Pages/homepage/homepage.component.scss]: There where two different weights for the fonts, the one from styles.css was unused, so I moved this one there and deleted the one that was too thick

**Review by vladcantor — APPROVED** [review]

**Pull request merged**